README.md

```diff
@@ -33,7 +33,12 @@ bin/kafka_bridge_run.sh --config-file config/application.properties
 
 ## Documentation
 
-Documentation to the current _main_ branch as well as all releases can be found on our [website](https://strimzi.io).
+Documentation can be found on our website:
+
+* for the [latest stable release](https://strimzi.io/docs/bridge/latest/)
+* for the "in development" [_main_ branch](https://strimzi.io/docs/bridge/in-development/)
```

Comment:
```suggestion
* [Documentation for the latest stable release](https://strimzi.io/docs/bridge/latest/)
* [Documentation that's _in development_ for the `main` branch](https://strimzi.io/docs/bridge/in-development/)
```

Reply:
Not for me. I think the link text for both describes what we're linking to. Maybe if we were saying more we wouldn't want to have a full sentence link, for example.